plugins/HHAnalyzer.cc

```diff
@@ -238,6 +238,8 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             ll.push_back(dilep); 
         }
     }
+    // have the ll collection sorted by ht
+    std::sort(ll.begin(), ll.end(), [&](HH::Dijet& a, HH::Dijet& b){return a.p4.Pt() > b.p4.Pt();});
```

Comment:
* I guess you meant Dilepton here
* I am not sure this is what we want : it gives the dileptons sorted by pt of the whole system instead of the dileptons sorted by the pt sum the two candidates. Here is a suggestion : 
    * https://github.com/BrieucF/HHAnalysis/blob/MIS/plugins/HHAnalyzer.cc#L663
    * https://github.com/BrieucF/HHAnalysis/blob/MIS/plugins/HHAnalyzer.cc#L700